pkg/cmd/cli/secrets/unlink_secret_from_object.go

```diff
@@ -88,6 +88,7 @@ func (o UnlinkSecretOptions) unlinkSecretsFromServiceAccount(serviceaccount *kap
 			if secretname == serviceaccount.Secrets[i].Name {
```

Comment:
If rmSecretNames is a string set, just use !rmSecretNames.Has(serviceaccount.Secrets[i].Name) instead of the inner loop and found check